front-end/.eslintrc.js

```diff
@@ -3,9 +3,7 @@ module.exports = {
   env: {
     node: true
   },
-  'extends': [
-    'plugin:vue/vue3-strongly-recommended',
-  ],
+  extends: ['plugin:vue/vue3-recommended', '@vue/prettier', 'eslint:recommended'],
   rules: {
     'no-console': 'off',
```

Comment:
Could you please format the whole file, remove all quotes? Since this file has been modified.